[2025-09-08T17:04:11.641+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:04:11.649+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskD scheduled__2025-08-09T00:00:00+00:00 [queued]>
[2025-09-08T17:04:11.654+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskD scheduled__2025-08-09T00:00:00+00:00 [queued]>
[2025-09-08T17:04:11.654+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:04:11.666+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): taskD> on 2025-08-09 00:00:00+00:00
[2025-09-08T17:04:11.669+0000] {standard_task_runner.py:72} INFO - Started process 17466 to run task
[2025-09-08T17:04:11.671+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskD', 'scheduled__2025-08-09T00:00:00+00:00', '--job-id', '123', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpnprzhgjx']
[2025-09-08T17:04:11.673+0000] {standard_task_runner.py:105} INFO - Job 123: Subtask taskD
[2025-09-08T17:04:11.702+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskD scheduled__2025-08-09T00:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:04:11.746+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskD' AIRFLOW_CTX_EXECUTION_DATE='2025-08-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-09T00:00:00+00:00'
[2025-09-08T17:04:11.746+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:04:11.757+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-08T17:04:11.758+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK D has executed!']
[2025-09-08T17:04:11.765+0000] {subprocess.py:86} INFO - Output:
[2025-09-08T17:04:11.766+0000] {subprocess.py:93} INFO - TASK D has executed!
[2025-09-08T17:04:11.766+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-08T17:04:11.780+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:04:11.780+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskD, run_id=scheduled__2025-08-09T00:00:00+00:00, execution_date=20250809T000000, start_date=20250908T170411, end_date=20250908T170411
[2025-09-08T17:04:11.805+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:04:11.813+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:04:11.814+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
